pre_commit_hooks/markdown_links.py

```diff
@@ -49,9 +49,17 @@ def _make_label(node: Any) -> str:
     return "".join(label)
 
 
-def _make_anchor(label: str, used_anchors: Dict[str, int]) -> str:
+def _make_anchor(node: Any, used_anchors: Dict[str, int]) -> str:
     """Chooses the appropriate anchor name for a header label."""
-    anchor = label.lower().strip()
+    # Pick out the text fragments that the anchor will be based off
```

Comment:
```suggestion
    # Pick out the text fragments that the anchor will be based off.
```

Minor punctuation nit (using this to make the edit)